### ISSUE TITLE
Incorrect values crafting a few items

### ISSUE THREAD

**Issue by Kylezzz101** [Secrets-of-Sosaria/World#40]
I tried to make bark, which makes scrolls. The recipe says 1 board to 1 bark, but it will make the whole stack of boards

I lost 10,000 boards in 1 attempt

same thing for Fuk darts

1 board for 1 dart, but it makes the whole stack

**Comment by EstelMatiazi** [comment]
Do you have multicrafting enabled on your server? I imagine you must have clicked in the 'craft 100' option or similar.

**Comment by kiri-e** [comment]
No, that's just how several items work without multicrafting enabled.
SetUseAllRes makes those recipes consume all of the resources to produce as many as it could because it would be a major pain to mass craft those recipes with the default system.
Sure, it might do something you don't expect every once in a while but the alternative is that it would take forever to craft them all one-by-one.
